projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.html

```diff
@@ -19,7 +19,7 @@
 </igx-pivot-header-row>
 
 <div igxGridBody (keydown.control.c)="copyHandler($event)" (copy)="copyHandler($event)" class="igx-grid__tbody" role="rowgroup">
-    <div class="igx-grid__tbody-content" tabindex="0" [attr.role]="dataView.length ? null : 'row'" (keydown)="navigation.handleNavigation($event)"
+    <div class="igx-grid__tbody-content" tabindex="0" [attr.role]="dataView.length ? null : 'row'" (keydown)="navigation.handleNavigation($event)" (focus)="navigation.focusTbody($event)"
```

Comment:
I see that there's a focus handler here but it does not seem to get hit when the first click in the grid is on one of the row header cells. This might be the reason that navigation with the arrow keys does not work if you directly click in the row headers and try to navigate.